src/snapred/backend/data/LocalDataService.py

```diff
@@ -731,16 +731,28 @@ def writeNormalizationState(self, normalization: Normalization):
     def readDetectorState(self, runId: str) -> DetectorState:
         detectorState = None
         pvFile = self._readPVFile(runId)
+        wav_value = None
+        wav_key_1 = "entry/DASlogs/BL3:Chop:Gbl:WavelengthReq/value"
+        wav_key_2 = "entry/DASlogs/BL3:Chop:Skf1:WavelengthUserReq/value"
+
```

Comment:
@darshdinger I added these lines to the version of next I'm testing pixel masking in. I think the code above is working (the state is identified from the input file) but I get an error because there is an additional function `detectorStateFromWorkspace` that the pixel masking is calling. This is also in `LocalDataService` L762 ish. This function has to be updated in a similar way to `readDetectorState`